notebooks/00_quick_start/sar_movielens.ipynb

```diff
@@ -59,9 +59,7 @@
     "import sys\n",
```

Comment:
<p>I guess removing the columns is cleaner, but I'm not sure if people will get confused when the input column names are not the default ones, maybe we should indicate it somehow, not sure if in the comments or put the columns names back. </p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/617/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
now the columns should be the default ones, (at least in terms of reco_utils.common.constants defaults). I felt that in the quickstart version, the less additional pieces that don't have material impact on the algorithm the better.

Reply:
yeah, that's reasonable.